.github/docker/Dockerfile.glibc

```diff
@@ -16,7 +16,7 @@ RUN apt-get -qq update && apt-get -qq install -y python3 build-essential && ldd
 RUN npm run install:libmongocrypt
 
 ARG RUN_TEST
-RUN [ -n "$RUN_TEST" ] && npm run test || echo 'skipping testing!'
+RUN if [ -n "$RUN_TEST" ]; then npm test ; else echo "skipping tests" ; fi
```

Comment:
This is a drive-by improvement.  I discovered in zstd that the current implementation squashes test failures.